### ISSUE TITLE
[VPAT] Multiple label errors in course settings view

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#4291]
There are multiple `<label>` errors in the course settings view ("A form control does not have a corresponding label").  

Many of these errors appear to be also associated with an orphaned `<label>` ("A form label is present, but it is not correctly associated with a form control").  Fixing an orphaned `<label>` may also fix an error.

For anyone who needs a quick refresher on the `<label>` tag, please check this [article at w3schools.com](https://www.w3schools.com/tags/tag_label.asp)

The screenshot shows a few of the errors, but there are 17 in total, of each, on the page.
![Screen Shot 2019-08-05 at 5 58 50 PM](https://user-images.githubusercontent.com/12703624/62498006-bc6c1000-b7ab-11e9-81c1-1d6dabc2a4e5.png)
